tools/flambda_backend_objinfo.ml

```diff
@@ -154,18 +154,34 @@ let print_global_table table =
 open Cmx_format
 open Cmxs_format
 
+let machtype_identifier t =
```

Comment:
We could, but I thought it was better to use a function that will not fail, even in case the input has `Addr` components (even though it shouldn't), in order to still be able to use `objinfo` in case we had a bug in the compiler and produced such objects.

Reply:
Maybe that function could go in `Cmm_helpers` then?  I'm just wary of having these pieces of code so far from each other; upon an update, we're bound to forget to edit the other place.

Reply:
I just reused the existing function actually, it seemed too dangerous to keep two almost identical versions, even if next to each other.